frontend/src/bundles/common/components/upload-video/components/dropzone/libs/helpers/is-file-size-valid.ts

```diff
@@ -0,0 +1,5 @@
+const isFileSizeValid = (file: File, maxSize: number): boolean => {
+    return file.size <= maxSize;
```

Comment:
Do we really need a separate helper function for this?

Reply:
why not?

Reply:
i think you can even move it to shared since we might use it on the backend also